test/e2e/http/actors.test.ts

```diff
@@ -84,6 +86,7 @@ describe("http/actors", () => {
     await server.actor.registerActor(DemoActorActivateImpl);
     await server.actor.registerActor(DemoActorTimerTtlImpl);
     await server.actor.registerActor(DemoActorReminderTtlImpl);
+    await server.actor.registerActor(DemoActorDeleteStateImpl);
```

Comment:
Can you please line 112 to update the number of registered actors? It should be 9 after this change.